test/integration/patients/patient/patient-flow.js

```diff
@@ -424,6 +424,12 @@ context('patient flow page', function() {
   });
 
   specify('add action', function() {
+    const currentClinician = getCurrentClinician({
```

Comment:
Can we flip these tests here and in patient-action to use the default `teamNurse` that the test `currentClinician` is by default so we don't need to set it?

Reply:
Good call, updated 👍